core/block/source/service.go

```diff
@@ -203,4 +204,21 @@ func (s *service) RemoveStaticSource(id string) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
 	delete(s.staticIds, id)
+	err := s.objectStore.DeleteDetails(id)
```

Comment:
Maybe it will be more reliable to query all objects with specific space id (i.e. marketplace) and remove them? Or create new key in badger for virtual objects' registry